README.md

```diff
@@ -1 +1,223 @@
+| zone | (Optional) The zone in which the created job should run. If it is not provided, the provider zone is used. | string | `"us-central1-a"` | no |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| df\_job\_id | The unique Id of the newly created Dataflow job |
+| df\_job\_name | The name of the dataflow job |
+| df\_job\_state | The state of the newly created Dataflow job |
+| df\_job\_template\_gcs\_path | The GCS path to the Dataflow job template. |
+| temp\_gcs\_location | The GCS path for the Dataflow job's temporary data. |
+
+[^]: (autogen_docs_end)
+
+## Requirements
+
+Before this module can be used on a project, you must ensure that the following pre-requisites are fulfilled:
+
+1. Terraform is [installed](#software-dependencies) on the machine where Terraform is executed.
+2. The Service Account you execute the module with has the right [permissions](#configure-a-service-account).
+3. The necessary APIs are [active](#enable-apis) on the project.
+4. A working Dataflow template in uploaded in a GCS bucket
+
+The [project factory](https://github.com/terraform-google-modules/terraform-google-project-factory) can be used to provision projects with the correct APIs active.
+
+### Software Dependencies
+### Terraform
+- [Terraform](https://www.terraform.io/downloads.html) 0.10.x
+- [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) plugin v1.8.0
+
+### Configure a Service Account to execute the module
+In order to execute this module you must have a Service Account with the
+following project roles:
+- roles/dataflow.admin
+- roles/iam.serviceAccountUser
+- roles/storage.admin
+
+### Configure a Controller Service Account to create the job
+If you want to use the service_account_email input to specify a service account that will identify the VMs in which the jobs are running, the service account will need the following project roles:
+- roles/dataflow.worker
+- roles/storage.objectAdmin
+
+### Enable APIs
+In order to launch a Dataflow Job, the Dataflow API must be enabled:
+
+- Dataflow API - dataflow.googleapis.com
+
+## Install
+
+### Terraform
+Be sure you have the correct Terraform version (0.10.x), you can choose the binary here:
+- https://releases.hashicorp.com/terraform/
+
+## File structure
+The project has the following folders and files:
+
+- /: root folder
+- /examples: examples for using this module
+- /helpers: Helper scripts
+- /test: Folders with files for testing the module (see Testing section on this file)
+- /main.tf: main file for this module, contains all the resources to create
+- /variables.tf: all the variables for the module
+- /output.tf: the outputs of the module
+- /README.md: this file
+
+## Testing
+
+### Requirements
+- [bundler](https://github.com/bundler/bundler)
+- [gcloud](https://cloud.google.com/sdk/install)
+- [terraform-docs](https://github.com/segmentio/terraform-docs/releases) 0.3.0
```

Comment:
```suggestion
- [terraform-docs](https://github.com/segmentio/terraform-docs/releases) 0.6.0
```

Reply:
Resolved in
https://github.com/terraform-google-modules/terraform-google-dataflow/pull/1/commits/e5535bdba8de2d82c01c6d62f54b757cb706932d